File: ezSCORES/ezSCORES.Model/Fixtures.cs
using ezSCORES.Model.ENUMs;
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.RegularExpressions;

namespace ezSCORES.Model
{
	public class Fixtures
	{
		public int Id { get; set; }

		public int CompetitionId { get; set; }

		public int SequenceNumber { get; set; }

		public int MatchLength { get; set; }

		public GameStage GameStage { get; set; }

		public bool IsCurrentlyActive { get; set; }

		public bool IsCompleted { get; set; }
		public virtual ICollection<Matches> Matches { get; set; } = new List<Matches>();
	}
}

File: ezSCORES/UI/ezscores_desktop/lib/models/fixtures.dart
import 'package:ezscores_desktop/models/enums/gameStage.dart';
import 'package:ezscores_desktop/models/matches.dart';
import 'package:json_annotation/json_annotation.dart';

part 'fixtures.g.dart';

@JsonSerializable()
class Fixtures {
	int? id;
	int? competitionId;
	int? sequenceNumber;
	int? matchLength;
	bool? isCurrentlyActive;
	bool? isCompleted;
  GameStage? gameStage;
  List<Matches>? matches;

	Fixtures();

	factory Fixtures.fromJson(Map<String, dynamic> json) => _$FixturesFromJson(json);

	Map<String, dynamic> toJson() => _$FixturesToJson(this);
}
